javacord-api/src/main/java/org/javacord/api/entity/activity/Activity.java

```diff
@@ -81,4 +81,9 @@ public interface Activity extends Nameable {
      */
     Optional<Emoji> getEmoji();
 
+    /**
+     * activity flags OR d together, describes what the payload includes
+     * @return the activity flags.
```

Comment:
```suggestion
     * Gets the activity flags of this activity.
     * The flags describe what the payload includes.
     *
     * @return the activity flags.
```
Also don't just copy paste the description straight from the docs. As you can see above we have a structure for the Javadoc and it should also follow this.